client/my-sites/patterns/controller.tsx

```diff
@@ -1 +1,17 @@
+import { addLocaleToPathLocaleInFront } from '@automattic/i18n-utils';
+import { PatternTypeFilter } from 'calypso/my-sites/patterns/types';
+
 export const RENDERER_SITE_ID = 226011606; // assemblerdemo
+
+export function getCategorySlug(
```

Comment:
What about `getPatternsCategoryUrl`?
Since actually it's not just slug, it's full path. Also it's a bit confusing to have function name `getCategorySlug` and argumeant as `categorySlug`.